modules/compute/gke-node-pool/variables.tf

```diff
@@ -255,6 +255,12 @@ variable "timeout_update" {
   default     = null
 }
 
+variable "user_workload_path" {
```

Comment:
Are we going to show the example usage of this in the example blueprints?

Reply:
updated